### PR TITLE
feat: add link fields for direct and indirect expenses in company master

### PR DESCRIPTION
**Pull request by rvpasari**
https://app.asana.com/0/1199693776410436/1199925128801509

### CODE REVIEW

**Comment by rvpasari** [comment]
Screen Shot 2021-02-11 at 7 17 56 PM

File: erpnext/setup/doctype/company/company.json [hunk 1]
```diff
@@ -736,13 +738,29 @@
    "fieldtype": "Link",
    "label": "Default Use Tax Payable Account",
    "options": "Account"
+  },
+  {
+   "depends_on": "eval:!doc.__islocal",
+   "fieldname": "default_direct_expenses",
+   "fieldtype": "Link",
+   "ignore_user_permissions": 1,
```
Comment (hrwX) [inline erpnext/setup/doctype/company/company.json]:
```suggestion
   "ignore_user_permissions": 1,
```

Why is there an ignore user permission ?

File: erpnext/setup/doctype/company/company.json [hunk 2]
```diff
@@ -736,13 +738,29 @@
    "fieldtype": "Link",
    "label": "Default Use Tax Payable Account",
    "options": "Account"
+  },
+  {
+   "depends_on": "eval:!doc.__islocal",
```
Comment (hrwX) [inline erpnext/setup/doctype/company/company.json]:
```suggestion
   "depends_on": "eval:!doc.__islocal",
```
any reason to hide it when new doc

**Review by hrwX — CHANGES_REQUESTED** [review]

File: erpnext/setup/doctype/company/company.json (hunk 2 above)
Comment (rvpasari) [reply erpnext/setup/doctype/company/company.json]: Not really, I had copied the configuration from the prior row in the site. I'll fix this.

**Review by sahil28297 — APPROVED** [review]

**Pull request merged**